i18n/Messages.properties

```diff
@@ -328,15 +328,16 @@ DXVK=DXVK
 #: Engines/Wine/Verbs/DXVK/script.js:41
 DXVK\ is\ currently\ unsupported\ on\ non-Linux\ operating\ systems\ due\ to\ MoltenVK\ implementation\ being\ incomplete.\ Select\ how\ do\ you\ want\ to\ approach\ this\ situation.=DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Select how do you want to approach this situation.
 
-#: Engines/Wine/Settings/GLSL/script.js:11 Engines/Wine/Settings/render target
-#: lock mode/script.js:11 Engines/Wine/Settings/DirectDraw
-#: renderer/script.js:11 Engines/Wine/Settings/video memory size/script.js:11
-#: Engines/Wine/Settings/mouse warp override/script.js:11
+#: Engines/Wine/Settings/offscreen rendering mode/script.js:11
```

Comment:
Can you revert the changes in this file? Not sure if it might cause problems.

Reply:
Done